### PR TITLE
Add back "automatic option" except for app

### PR DESCRIPTION
**Pull request by ovbm**

### CODE REVIEW

File: components/Frame/DarkmodeSwitch.js [hunk 1]
```diff
@@ -67,18 +71,20 @@ const DarkmodeSwitch = ({ colorSchemeKey: pageColorSchemeKey, t }) => {
...
-              }}
-            >
-              {t('darkmode/switch/auto')}
-            </Radio>*/}
+            {!inNativeApp ? (
+              <Radio
+                checked={!colorSchemeKey}
+                onChange={() => {
+                  // default is undefined
+                  setColorSchemeKey()
```
Comment (tpreusse) [inline components/Frame/DarkmodeSwitch.js:82]:
```suggestion
                  setColorSchemeKey('auto')
```

File: components/Frame/DarkmodeSwitch.js [hunk 2]
```diff
@@ -67,18 +71,20 @@ const DarkmodeSwitch = ({ colorSchemeKey: pageColorSchemeKey, t }) => {
...
-              onChange={() => {
-                // default is undefined
-                setColorSchemeKey()
-              }}
-            >
-              {t('darkmode/switch/auto')}
-            </Radio>*/}
+            {!inNativeApp ? (
+              <Radio
+                checked={!colorSchemeKey}
```
Comment (tpreusse) [inline components/Frame/DarkmodeSwitch.js:79]:
```suggestion
                value='auto'
                checked={!colorSchemeKey}
```

File: components/Frame/DarkmodeSwitch.js [hunk 3]
```diff
@@ -67,18 +71,20 @@ const DarkmodeSwitch = ({ colorSchemeKey: pageColorSchemeKey, t }) => {
...
+                onChange={() => {
+                  // default is undefined
```
Comment (tpreusse) [inline components/Frame/DarkmodeSwitch.js:81]:
```suggestion
                  // ToDo activating auto by default
                  // - handle all «ToDo activating auto» comments
                  // - rm explicit auto value
```

File: components/Frame/DarkmodeSwitch.js (hunk 1 above)
Comment (ovbm) [reply components/Frame/DarkmodeSwitch.js:82]: shouldn't it still be default white, with just the option to set it as auto?

Comment (tpreusse) [reply components/Frame/DarkmodeSwitch.js:82]: Well here we need to explicitly save `auto` into the local storage.

When there is no local storage—or a empty value—the frontend is configured to activate light (via color synch & head script)

Comment (ovbm) [reply components/Frame/DarkmodeSwitch.js:82]: ah logo...

**Review by tpreusse — APPROVED** [review]

**Pull request merged**